server/Arcadia.Assistant.Web/Controllers/UserController.cs

```diff
@@ -39,5 +51,33 @@ public UserController(IUserEmployeeSearch userEmployeeSearch)
                     Username = userEmployee.Metadata.Name
                 });
         }
+
+        [Route("permissions")]
+        [HttpGet]
+        [ProducesResponseType(typeof(UserEmployeePermissionsModel), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetPermissions(CancellationToken token)
```

Comment:
an error, why do you load permissions for yourself? You need to pass an employeeId and load current user permission for this employee

Reply:
to see only self permissions. Do we need if someone can see permissions of others employees?